src/content/en/fundamentals/performance/critical-rendering-path/measure-crp.md

```diff
@@ -5,20 +5,51 @@ description: Learn to measure the critical rendering path.
 {# wf_updated_on: 2014-09-17 #}
 {# wf_published_on: 2014-03-31 #}
 
-# Measuring the Critical Rendering Path with Navigation Timing {: .page-title }
+# Measuring the Critical Rendering Path {: .page-title }
 
 {% include "web/_shared/contributors/ilyagrigorik.html" %}
 
-You can't optimize what you can't measure. Thankfully, the Navigation 
-Timing API gives us all the necessary tools to measure each step of the 
-critical rendering path!
+The foundation of every solid performance strategy is good measurement and
+instrumentation. You can't optimize what you can't measure. This doc
+explains different approaches for measuring CRP performance.
 
-### TL;DR {: .hide-from-toc }
-- Navigation Timing provides high resolution timestamps for measuring CRP.
-- Browser emits series of consumable events which capture various stages of the CRP.
+The key difference between the different approaches is the different kind
+of metrics that they can capture.
 
+* The Lighthouse approach runs a series of automated tests against a page,
+  and then generates a report on the page's CRP performance. This approach
+  provides a quick and easy high-level overview of CRP performance, but does
+  not truly reflect how users experience the page.
+* The Navigation Timing API approach captures [Real User
+  Monitoring (RUM)](https://en.wikipedia.org/wiki/Real_user_monitoring)
+  metrics. As the name implies, these metrics are captured from real user
+  interactions with your site. This approach provides a more accurate view
+  of CRP performance, but is more difficult and time-consuming to set up.
 
-The foundation of every solid performance strategy is good measurement and instrumentation. Turns out, that is exactly what the Navigation Timing API provides.
+In general, a good approach is to use Lighthouse to identify obvious CRP
+optimization opportunities, and then to instrument your code with the
+Navigation Timing API to truly measure how your app performs out in the wild.
+
+## Auditing a page with Lighthouse {: #lighthouse }
+
+Lighthouse is a web app auditing tool that runs a series of tests against a
+given page, and then displays the page's results in a consolidated report. You
+can run Lighthouse as a Chrome Extension or NPM module (which is
+useful for integrating Lighthouse with continuous integration systems).
+
+Lighthouse has a suite of audits for measuring a page's CRP. When you
+run Lighthouse as a Chrome Extension, your page's CRP results look like the
+screenshot below.
```

Comment:
I added a link to the audit's reference doc, which is merged and should be live soon. We can wait until then before merging this.

Here's the reference doc source which you can review in the meantime: https://github.com/google/WebFundamentals/pull/3693/files#diff-45fb923d6ae73708ae68413d31216d4c